llama_index/indices/postprocessor/types.py

```diff
@@ -1,16 +1,27 @@
+import asyncio
 from abc import ABC, abstractmethod
 from typing import List, Optional
 
 from llama_index.indices.query.schema import QueryBundle
 from llama_index.schema import NodeWithScore
+from llama_index.async_utils import run_sync
 
 
 class BaseNodePostprocessor(ABC):
-    @abstractmethod
+
     def postprocess_nodes(
         self,
         nodes: List[NodeWithScore],
         query_bundle: Optional[QueryBundle] = None,
+    ) -> List[NodeWithScore]:
+        """Postprocess nodes."""
+        return run_sync(self.apostprocess_nodes(nodes, query_bundle))
+
+    @abstractmethod
```

Comment:
I'm a little torn on making this an abstract method. Right now it's really easy UX for users to extend this class for their own post-processors

Reply:
Hmm, wouldn't it be the same as the current UX but now they're defining it as an async method? Also should be backwards compatible if they previously had their logic in the synchronous `postprocess_nodes` method.

Reply:
Yea I guess they just have to add an async? But it will definitely break in a weird way for existing users, since they will have the sync method implemented and overridden, and then they will get an error about a missing abstract method  (just more-so making that known)